### PR TITLE
Fix columns in Felix wireguard documentation

### PR DESCRIPTION
**Pull request by sms-system**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Comment by lwr20** [comment]
@ctauchen Could you review this PR please?

**Review by ctauchen — APPROVED** [review]
@lwr20 This PR corrects a misformatted table:
* The headers 'Environment variable', 'Description', and 'Schema' apply to different columns, and they appear to be correct.
* A 'Default' header was added and applied to one of the existing columns.
* Data was added to the `Environment variable` column. This information is new.

This appears to be a good fix. LGTM, pending SME review.

@lwr20 Can you review the technical details? Or suggest someone?

If we can't get a tech review by Monday, I'll take this over and get it to the new repo.



Environment variable | Description | Schema
-- | -- | --

**Review by mgleung — APPROVED** [review]

**Pull request merged**
